shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -464,7 +464,7 @@
       return;
     }
     fml::scoped_nsobject<FlutterOverlayView> overlay_view(
-        [[[FlutterOverlayView alloc] init] retain]);
+        [[FlutterOverlayView alloc] init]);
```

Comment:
this is not safe - scoped_nsobject does not automatically retain the object and you could end up with a dead pointer here.